device/api/umd/device/cluster.h

```diff
@@ -13,14 +13,14 @@
 #include <unordered_set>
```

Comment:
You don't need to change anything and this remark isn't specific to this file...

device/cluster.h -> device/api/umd/device/cluster.h

Can we declutter this?  Get rid of "device"?  Now it's in there twice

Reply:
Yes, that will happen. Also it might make sense to remove "device" altogether, since "umd" itself is "device"
https://github.com/tenstorrent/tt-umd/issues/324

Reply:
related: I've never liked the library name "libdevice.so" ... could there be a more generic name? However, it may be too late to fix.

vote: `libtt_umd.so`